content/docs/12-enterprise-version/06-air-gap-repo.md

```diff
@@ -1,108 +1,730 @@
+
+* Ensure the following ports allow inbound network traffic. 
     * 80
     * 443
     * 5000
     * 8000
 
 
-<InfoBox>
-Storage sizing depends on your intended update frequency and data retention model.
-</InfoBox>
+* Request the Palette self-hosted installer image and the Palette air gap installer image. To request the installer images, please contact our support team by sending an email to support@spectrocloud.com. Kindly provide the following information in your email:
+
+    - Your full name
+    - Organization name (if applicable)
+    - Email address
+    - Phone number (optional)
+    - A brief description of your intended use for the Palette Self-host installer image.
+
+Our dedicated support team will promptly get in touch with you to provide the necessary assistance and share the installer image. 
+
+If you have any questions or concerns, please feel free to contact support@spectrocloud.com.
+
+<br />
+
+
+# Deploy Air Gapped Appliance
+
+
+1. Log in to vCenter Server by using the vSphere Client.
+
+
+2. Navigate to the Datacenter and select the cluster you want to use for the installation. Right-click on the cluster and select **Deploy OVF Template**.
+
+
+3. Select the airgap OVA installer image you downloaded after receiving guidance from our support team.
+
+
+4. Select the folder where you want to install the Virtual Machine (VM) and assign a name to the VM.
+
+
+5. Next, select the compute resource.
+
+
+6. Review the details page. You may get a warning message stating the certificate is not trusted, you can ignore this message and select **Next**.
+
+
+7. Select your storage device and storage policy. Click on **Next** to proceed.
+
+
+8. Choose a network for your appliance and select **Next**.
+
+
+9. Fill out the remaining template customization options. You can modify the following input fields. <br /><br />
+
+  | Parameter | Description |  Default Value |
+  | --- | --- | -- |
+  | Encoded user-data | In order to fit into a xml attribute, this value is base64 encoded . This value will be decoded, and then processed normally as user-data. | - |
+  | ssh public keys | This field is optional, but indicates that the instance should populate the default user's `authorized_keys` with this publick key provided | -|
+  | Default User's password|  If set, the default user's password will be set to this value to allow password based login.  The password will be good for only a single login.  If set to the string `RANDOM` then a random password will be generated, and written to the console. | - |
+  | A Unique Instance ID for this instance|  Specifies the instance id. This is required and used to determine if the machine should take "first boot" actions| `id-ovf`|
+  |hostname| Specifies the hostname for the appliance. | `ubuntuguest` |
+  |Url to seed instance data from| This field is optional, but indicates that the instance should 'seed' user-data and meta-data from the given url.| -|
+
```

Comment:
Should parameter names be boldfaced (since they're in a form). I boldfaced parm names in the QS doc.